#!/usr/bin/env python
"""
"""

from dynamic_reconfigure.parameter_generator_catkin import *

PACKAGE = "sphero_formation"

gen = ParameterGenerator()

gen.add("test_duration", int_t, 0, "Test duration", 10, 0, 30)
gen.add("setpoint_speed", double_t, 0, "Setpoint speed", 0.5, 0, 1)
gen.add("heading_change", int_t, 0, "Heading change", 90, 0, 180)
gen.add("heading_time", int_t, 0, "Heading change moment", 5, 0, 30)
gen.add("turning_rate", int_t, 0, "Max turning rate", 90, 0, 180)



exit(gen.generate(PACKAGE, "test_reconfigure_node", "Test"))
